libraries/logging/stream-logger-android/detekt-baseline.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<SmellBaseline>
+  <ManuallySuppressedIssues/>
+  <CurrentIssues>
+    <ID>EmptyFunctionBlock:ActivityLifecycleCallbacks.kt$ActivityLifecycleCallbacks${}</ID>
+    <ID>SwallowedException:StreamLogFileProvider.kt$StreamLogFileProvider$e: PackageManager.NameNotFoundException</ID>
```

Comment:
Can we fix those issues before merging?

Reply:
Those 2 are produced because of the following things:

1. **EmptyFunctionBlock**
```kotlin
internal abstract class ActivityLifecycleCallbacks : Application.ActivityLifecycleCallbacks {

    override fun onActivityCreated(activity: Activity, bunlde: Bundle?) {}

    override fun onActivityStarted(activity: Activity) {}

    override fun onActivityResumed(activity: Activity) {}

    override fun onActivityPaused(activity: Activity) {}

    override fun onActivityStopped(activity: Activity) {}

    override fun onActivitySaveInstanceState(activity: Activity, bunlde: Bundle) {}

    override fun onActivityDestroyed(activity: Activity) {}
}
```

2. **SwallowedException**
```
private fun Context.getVersionName(): String = try {
    packageManager?.getPackageInfo(packageName, 0)?.versionName ?: ""
} catch (e: PackageManager.NameNotFoundException) {
    ""
}
```

Reply:
If we really need those empty methods (Maybe we don't really need this empty class and move the implementation to the class that implement this abstract class), we should add a comment like `// no-op`, with it we are sure it was thought to not do anything and it is not because we forgot to implement it

For the second one, as far as we are not using the exception, it can be called `_` (default Kotlin's name for ignored arguments) or contain `ignored` as a part of the name

Reply:
It is ok you create the `ActivityLifecycleCallbacks` class, but it needs to follow our Detek Rules